webapp-django/crashstats/home/static/home/js/home.js

```diff
@@ -0,0 +1,386 @@
+/*global MG: true, socorro: true */
+
+$(function () {
+    'use strict';
+
+    // A polyfill for `endsWith`
+    if (!String.prototype.endsWith) {
```

Comment:
@peterbe What do you think about polyfills in Socorro? I guess you don't mind them since you didn't comment here. But should we maybe centralize them? I could see us having a file called ``polyfills.js`` that we would include on every page, and which would contain all of our polyfills. That way it's easy to maintain by regularly verifying if something can be removed.